### ISSUE TITLE
Error: Could not find enough samples for a reliable detection.

### ISSUE THREAD

**Issue by CaiIsProgrammer** [dlepaux/realtime-bpm-analyzer#34]
I'm trying to connect it to a live radio stream but it just keeps throwing that error. is there a way to do this with a live feed?
```
  getBPM = () => {
    console.log("bpm");
    let { source } = this.state;
    let audioContext = this.props.audioContext;
    const scriptProcessorNode = audioContext.createScriptProcessor(4096, 1, 1);
    scriptProcessorNode.connect(audioContext.destination);
    source.connect(scriptProcessorNode);

    const onAudioProcess = new RealTimeBPMAnalyzer({
      debug: true,
      scriptNode: {
        bufferSize: 4096,
        numberOfInputChannels: 1,
        numberOfOutputChannels: 1,
      },
      pushTime: 1000,
      pushCallback: (err, bpm) => {
        console.log("65", err);
        console.log("bpm", bpm);
      },
    });
    scriptProcessorNode.onaudioprocess = (e) => {
      onAudioProcess.analyze(e);
    };
  };
```

**Comment by dlepaux** [comment]
Hello,

Could you provide more informations about your issue please?

**Comment by CaiIsProgrammer** [comment]
Yea, so what I'm trying to do is take a live radio url and finding the current bpm of a certain song. Do you know how i could set it up?

**Comment by dlepaux** [comment]
Did you saw this implementation ? 
https://github.com/dlepaux/realtime-bpm-analyzer-exemple/blob/master/app/js/userMedia.js
I believe that it could fits your needs !
Keep me in touch

**Comment by CaiIsProgrammer** [comment]
so actually its not user media 

`
    let url = this.state.radio;
    audioFile = new Audio();
    audioFile.crossOrigin = "anonymous";
    audioFile.src = url;

    //play audio
    audioFile.load();
    audioFile.volume = this.state.volume / 100.0;

    let playPromise = audioFile.play();

    if (playPromise !== undefined) {
      playPromise
        .then((_) => {
          // Automatic playback started!

          source = this.props.audioContext.createMediaElementSource(audioFile);
          source.connect(this.props.audioContext.destination);

          this.setState({
            audioFile,
            source,
            playing: true,
          });
        })
        .catch((error) => {
          // Auto-play was prevented
          // Show paused UI.
          console.log("playback prevented", error);
        });
    }



` but it is an audio stream

**Comment by CaiIsProgrammer** [comment]
do you have discord for a call?

**Comment by CaiIsProgrammer** [comment]
if so add me KingCairo#0069

**Comment by dlepaux** [comment]
UserMedia behave like a stream actually.
Do you have a repository with your code or something ? I can't help you if I can't see you code.

**Comment by dlepaux** [comment]
FYI I've implemented a solution with the new unreleased version of the library.
I will keep you in touch when it will be.

**Comment by dlepaux** [comment]
I just released the version 2.1.0 (https://www.npmjs.com/package/realtime-bpm-analyzer/v/2.1.0) :)
